src/View/View.php

```diff
@@ -1,16 +1,16 @@
 <?php
 /**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
```

Comment:
Yes i think a separate commit is good for this work.Because of almost all of file's url cakephp.org has this problem.